lib/hydra/works/models/concerns/generic_file_behavior.rb

```diff
@@ -0,0 +1,44 @@
+module Hydra::Works
+  module GenericFileBehavior
```

Comment:
I think that terminology will be confusing. a Works::File is a PCDM::Object and not a PCDM::File.